tests/unit/components/link-to-username-test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  moduleForComponent,
+  test
+} from 'ember-qunit';
+
+moduleForComponent('link-to-username', {
+  // Specify the other units that are required for this test
+  // needs: ['component:foo', 'helper:bar']
+  unit: true
+});
+
+test('role returns the IRC role of the user', function(assert) {
```

Comment:
@galfert Should we merge this, so it's ready to be styled tho? We could create a seperate issue for making this feature IRC-only (or rather: add the tag based on the protocol's API).